ci/ci-common.yml

```diff
@@ -30,7 +30,7 @@ stages:
     reports:
       dotenv: build.env
   variables:
-    SPACK_SHA: develop-2024-12-15
+    SPACK_SHA: 2edbed3a9db73efed01c37037cb2bfd499f5b569
     SPACK_BUILDCACHE: develop-2024-12-15
```

Comment:
I suppose you might be able to bump this to a `develop-2025-???` tag if you like?